### ISSUE TITLE
Package names when using multipackage

### ISSUE THREAD

**Issue by pelletiermaxime** [lamont-cookbooks/multipackage#1]
Hi,  first of all, thanks for the cookbook.

When using multipackage_install %w(php5 php5-mysql) everything works fine. If I change to using multipackage %w(php5 php5-mysql) it seems to not properly parse the array and it tries to install the packages "php5," and "php5-mysql" :

```
Ran apt-get -q -y install php5, php5-mysql
Error executing action `install` on resource 'multipackage[php5, php5-mysql]'
```

Thanks.

**Comment by spheromak** [comment]
we also ran into this today as well.

**Comment by lamont-granquist** [comment]
fixed in 3.0.20

note that in 2.x -> 3.x  the multipackage resource got renamed to multipackage_internal.  `multipackage_install` should be rewritten just as `multipackage` now.  the `multipackage_install` definition is still there for backcompat.
